core/spec.md

```diff
@@ -1117,6 +1122,47 @@ HTTP body.
 
 ---
 
+### Registry Well-Known Config
+
+The registry comes with a well known location `/.well-known/xregistry.json`,
+following [RFC8615](https://datatracker.ietf.org/doc/html/rfc8615).
+
+It acts as a single entrypoint from where locations and capabilities of the
+xRegistry implementation can be retrieved. Since this URL is always the same,
+it can be used to find out whether xRegistry is supported and where to find the API.
+It also allows to support multiple versions of the xRegistry protocol at the same time
+and allows implementers to choose their own API base path.
+
+```yaml
+{
+  # The following top-level attributes are identical to the xRegistry API root response
+  # We may consider removing them here if we're bothered by the duplication.
+  "id": "STRING",
+  "name": "STRING", ?
+  "epoch": UINTEGER,
+  "self": "URL",
+  "description": "STRING", ?
+  "documentation": "URL", ?
+  "labels": { "STRING": "STRING" * }, ?
+  "createdat": "TIME",
+  "modifiedat": "TIME",
+
+  # Describe where to find the xRegistry API and its capabilities
+  "apis": {
```

Comment:
Good point. First I wanted to make this an array, but then I remembered that you often preferred to have dictionaries. But since you don't need to have links to this, I would then propose something like:

```ts
apis: Array<{ version: "v1" | "v2", url: "URL" }>
```